docs/release-notes/5.7.44-51.md

```diff
@@ -0,0 +1,63 @@
+# Percona Server for MySQL 5.7.44-51 (2024-08-00)
+
+This release is part of {{post}}. The fixes are available to [MySQL 5.7 Post-EOL Support from Percona customers]. Community members can [build this release from the source]. We recommend that you upgrade to MySQL 8.0 or you can stay on 5.7; we'll support you either way.
+
+## Release Information
+
+Percona Server for MySQL 5.7.44-51 builds upon the functionality and bug fixes in MySQL 5.7.44 Community Edition. It adds enterprise-grade features and security enhancements developed by Percona.
+
+
+Percona Server for MySQL 5.7.44-51 contains the following fixes.
+
+## Bug fixes
+
+### Security fix
+
+* [Bug #36563773]: The calculation of averages for some numbers was sometimes incorrect.
+
+### Non-security fixes
+
+* [Bug #35205358]: The `conn_delay/Waiting` stage in the connection_control plugin was not reset after a delay caused by the plugin, leading to inaccurate monitoring data.
+
+* [Bug #34929814]: Running two OPTIMIZE TABLE commands at the same time on a table with fulltext indexes and innodb_optimize_fulltext_only enabled sometimes made the server shut down.
+
+* [Bug #35894664]: The result from a spatial index that included a column with a spatial reference identifier (SRID) was empty. Additionally, using FORCE INDEX to enforce a covering index scan on the spatial index caused an assertion error.
+
+* [Bug #331426544, #36543458]: Adds an additional check on the QT_NO_TABLE flag before accessing table_name.
```

Comment:
There are too bugs. So maybe change to 

[`Bugs` #331426544, #36543458]: `Add` an additional check on the QT_NO_TABLE flag before accessing table_name.